modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -3145,12 +3145,79 @@ definition {
 	}
 
 	@description = "LPS-135390 - Verify it is possible to update an Object with the Update permission"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanUpdateObjectWithUpdatePermission {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Custom Field",
+			fieldName = "customObjectField",
+			fieldType = "String",
+			isRequired = "false",
+			objectName = "CustomObject");
+
+		JSONRole.addRegularRole(
+			roleKey = "Regular Role",
+			roleTitle = "Regular Role");
+
+		for (var resourceAction : list "ACCESS_IN_CONTROL_PANEL,VIEW") {
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "${resourceAction}",
+				resourceName = "com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet",
+				roleTitle = "Regular Role");
+		}
+
+		for (var resourceAction : list "UPDATE,VIEW") {
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "${resourceAction}",
+				resourceName = "com.liferay.object.model.ObjectDefinition",
+				roleTitle = "Regular Role");
+		}
+
+		JSONUser.addUser(
+			userEmailAddress = "userea@liferay.com",
+			userFirstName = "userfn",
+			userLastName = "userln",
+			userScreenName = "usersn");
+
+		JSONUser.setFirstPassword(
+			agreeToTermsAndAnswerReminderQuery = "true",
+			requireReset = "false",
+			userEmailAddress = "userea@liferay.com");
+
+		JSONRole.assignRoleToUser(
+			roleTitle = "Regular Role",
+			userEmailAddress = "userea@liferay.com");
+
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "userea@liferay.com",
+			userLoginFullName = "userfn userln");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+		ObjectAdmin.editCustomObject(
+			fieldLabelName = "Object Edit",
+			fieldName = "ObjectEdit",
+			pluralLabelName = "Objects Edit");
+
+		CreateObject.selectTitleField(fieldLabel = "Custom Field");
+
+		ObjectAdmin.selectObjectScope(fieldOption = "Company");
+
+		CreateObject.selectPanelCategoryKey(panelCategoryKey = "Control Panel > Object");
+
+		CreateObject.saveObject();
+
+		ObjectAdmin.openObjectAdmin();
 
-		// TODO LPS-141606 CanUpdateObjectWithUpdatePermission pending implementation
+		ObjectPortlet.viewEntry(entry = "company");
 
+		ObjectAdmin.viewCustomObject(name = "Object Edit");
```

Comment:
Hi @andreefarias 

You can validate that the Title Field and Panel Category Key are updated too?